src/lib/turtle.js

```diff
@@ -1321,7 +1321,8 @@ function generateTurtleModule(_target) {
             }
         };
         proto.$register_shape.minArgs = 1;
-
+        proto.$register_shape.co_varnames = ["name", "points"];
+        
```

Comment:
looking at the definition in cpython - isn't the second argument called shape?

Reply:
maxArgs is not necessary, the defaultValue of maxArgs is the func's length